docs/tutorials/fundamentals/tutorial-token-transfer-and-nft-purchase.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Tutorial - Token transfer and Uniq purchase transactions'
-order: -99998
+order: -99997
```

Comment:
Better put it as -99996, the -99997 is used by `Tutorial - Log in to the Ultra Toolkit`